feat: Organize global admin settings form into tabs for better UX

SxxAq opened the pull request:
## Description
This PR implements the redesign of Global Admin Settings to align with the event settings UI pattern. The new design incorporates a tabbed interface that improves organization and user experience for administrators.
### Changes Made

- Restructured the Global Admin Settings interface to use a tabbed layout

- Organized settings into five distinct categories:

        - Basics
        - Localization
        - Email
        - Payment Gateways
        - Maps
### Screenshots
![250323_00h41m38s_screenshot](https://github.com/user-attachments/assets/e7a96d88-86d7-42c4-982f-45a5ba88d859)
![250323_00h42m29s_screenshot](https://github.com/user-attachments/assets/fd8ed452-bcb0-4ea8-9deb-9b801b97336e)
![250323_00h42m21s_screenshot](https://github.com/user-attachments/assets/05ba63ab-d88b-4693-82cf-747890c378b3)
![250323_00h42m14s_screenshot](https://github.com/user-attachments/assets/8d5aad64-b619-4e5a-8fdd-4efa03180d25)
![250323_00h42m03s_screenshot](https://github.com/user-attachments/assets/d2d1f94f-eb89-4e86-8aa3-40f2916ccc99)

### Related Issue
🎯 Closes #479

## Summary by Sourcery

Organize the global admin settings form into tabs for better user experience. This change restructures the global admin settings interface to use a tabbed layout, organizing settings into five distinct categories: Basics, Localization, Email, Payment Gateways, and Maps.

Enhancements:
- Restructure the Global Admin Settings interface to use a tabbed layout to improve organization and user experience.
- Organize settings into five distinct categories: Basics, Localization, Email, Payment Gateways, and Maps.

norbusan commented on src/pretix/control/forms/global_settings.py:
```diff
@@ -13,215 +13,343 @@
+        if (
+            global_settings.get("smtp_port") is None
+            or global_settings.get("smtp_port") == ""
+        ):
```
You seem to have a different line length setting, which introduces a lot of unrelated changes.
Please change this and make sure that no code changes are mixed with reformatting issues.

norbusan commented on src/pretix/control/forms/global_settings.py:
```diff
@@ -13,215 +13,343 @@
+                ),
+                (
+                    "banner_message",
+                    I18nFormField(
```
Same in all the changes here, unrelated reformatting changes

norbusan reviewed (changes requested):
As mentioned in the two comments, please separate code changes from reformatting changes.
There are **MANY** places where code was reformatted.

SxxAq commented:
@norbusan I have resolved the formatting-related issues. Please review the changes.

norbusan commented:
> @norbusan I have resolved the formatting-related issues. Please review the changes.

No you have not. Most of the changes are still unrelated reformatting changes, like splitting into multiple lines, changing the quote style from single to double, etc etc etc.

I did not mark all occurrences, only two or three. But this doesn't mean there aren't any other..

Please look yourself through the diff and check whether a change was made by you or bya reformatter.

mariobehling commented:
@SxxAq Please also improve the following.

Please compare the UI of other areas like the Payment Settings.

![Screenshot from 2025-04-24 18-01-20](https://github.com/user-attachments/assets/abaa5f5f-137f-4ceb-b950-28273226b0f4)

Do you see that the gray surrounding line is connected to the submenu? I understand that this design might break if there are too many items in the menu, but we only have a few. Therefore please implement it in the same way. Thank you!

SxxAq commented:
> @SxxAq Please also improve the following.
> 
> Please compare the UI of other areas like the Payment Settings.
> 
> ![Screenshot from 2025-04-24 18-01-20](https://private-user-images.githubusercontent.com/1583873/437108476-abaa5f5f-137f-4ceb-b950-28273226b0f4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDU1MjMyMTAsIm5iZiI6MTc0NTUyMjkxMCwicGF0aCI6Ii8xNTgzODczLzQzNzEwODQ3Ni1hYmFhNWY1Zi0xMzdmLTRjZWItYjk1MC0yODI3MzIyNmIwZjQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDQyNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTA0MjRUMTkyODMwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MTY0MDQzYmY1ZTlkMTIxMmZmNDFkZDQ3ZTc3NmI0OTU2ZTAyZjIyYTU2NWY5NDU4YzM3ZDQ1OGMwZjdmYjVlYiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.TlB-b8DY6rbkiMu5X-3NhuQH91yiZn274x8FOUbY9Ts)
> 
> Do you see that the gray surrounding line is connected to the submenu? I understand that this design might break if there are too many items in the menu, but we only have a few. Therefore please implement it in the same way. Thank you!

sure, I will implemet the suggested UI enhancement and undo the reformatting changes.

SxxAq commented:
![250426_22h48m22s_screenshot](https://github.com/user-attachments/assets/1a0a874f-65cd-471b-9605-18688a4ea423)
![250426_22h48m06s_screenshot](https://github.com/user-attachments/assets/c405be8d-982a-4a6f-ae15-58a915ecc2d2)

@mariobehling @norbusan : gray surrounding line connected to submenu as suggested. also undoed the reformatting changes.

SxxAq commented:
> Can you please look yourself through the changes?

Hey @norbusan,
I’ve manually reviewed all the files I modified, and I’ve corrected all the unrelated formatting changes. I cross-verified everything, and it looks good to me now. Could you please take a look and review?

Also, do you think I should squash all the commits into a single clean commit?

norbusan reviewed (approved):
LGTM, thanks for fixing the reformatting issues.

hongquan commented:
Look good enough.
A minor issue is that you added jQuery-based JS code. One of our goals is to migrate from jQuery to Vue, so we should not add more jQuery code. But it is OK for this code to stay in this PR because it simple enough to be converted later.

mariobehling commented:
@SxxAq Thanks for working on this! Please help to finalize this. See my comment here https://github.com/fossasia/eventyay-tickets/issues/479#issuecomment-2834477160

mariobehling commented:
@SxxAq I had to revert it as Paypal and other payment options were missing which I only realized now.

![Screenshot from 2025-05-01 21-00-35](https://github.com/user-attachments/assets/9ccb3698-83b8-4a40-b35f-bf6a88ef5ecb)

SxxAq commented:
> @SxxAq I had to revert it as Paypal and other payment options were missing which I only realized now.

hey @mariobehling I checked the global_settings related configs and i couldnt find anything related to paypal there??

SxxAq commented:
> > @SxxAq I had to revert it as Paypal and other payment options were missing which I only realized now.
> 
> hey @mariobehling I checked the global_settings related configs and i couldnt find anything related to paypal there??

![image](https://github.com/user-attachments/assets/92337167-8149-495a-b00e-fddd208ee33b)

mariobehling commented:
@SxxAq There are two plugins which are included in the deployment. They are:
https://github.com/fossasia/eventyay-tickets-paypal
https://github.com/fossasia/eventyay-tickets-stripe

These two plugins also need to be adjusted as part of solving this. Would be great if you come back to help with this.

mariobehling commented:
The paypal and stripe plugin are installed on the test system on app-test.eventyay.com, but if you install on your localhost depends on your setup.

Plugins:
https://github.com/fossasia/eventyay-tickets-paypal
https://github.com/fossasia/eventyay-tickets-stripe

SxxAq commented:
![439695824-9ccb3698-83b8-4a40-b35f-bf6a88ef5ecb](https://github.com/user-attachments/assets/cba1a486-13b1-4132-aed3-921934f05c09)


@norbusan @hongquan  how do i get all the payment gateway settings in my development setup? by default its only showing those 5 stripe related settings. is this a plugin issue? if so then how do i configure the plugins to work with the ticket component coz i tried setting up the plugin manually but it didnt work.
